utils/codegen/asn1ToC/asn1ToC.py

```diff
@@ -54,9 +56,20 @@ def parseCli():
 
     return args
 
+def process_file(args):
```

Comment:
Rename to `def adjustIncludesInFile` and add minimal docstring and perhaps some comments